lib/common-yargs.js

```diff
@@ -75,6 +74,13 @@ const withTitle = yargs => yargs.option("title", {
 	type: "string",
 })
 
+const withAccountType = yargs => yargs.option("account_type", {
```

Comment:
`account_type` is hyphenated. Naff. Better would be `--account-type`.

This is also a move away from having `--user`, `--repo`, or `--org`. Any reasoning for that?